core/bn254/attestation.go

```diff
@@ -95,59 +94,31 @@ func MulByGeneratorG2(a *fr.Element) *bn254.G2Affine {
 }
 
 func SerializeG1(p *bn254.G1Affine) []byte {
```

Comment:
Wouldn't it be better to just discard these altogether, and just make use of the gnark methods whenever the serialization methods are called?

Reply:
+1

Reply:
fixed